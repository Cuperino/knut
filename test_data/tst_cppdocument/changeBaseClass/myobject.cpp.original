#include "myobject.h"

#include <iostream>


MyObject::MyObject()
    : CPropertyPage()
{}

MyObject::MyObject(const QString &msg)
    : CPropertyPage(msg)
{}

MyObject::~MyObject()
{
    CPropertyPage::reset();
}


void MyObject::sayMessage(const std::string& test) {
    std::cout << test << std::endl;
    CPropertyPage::bla();
}
