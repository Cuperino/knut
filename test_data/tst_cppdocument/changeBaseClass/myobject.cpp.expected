#include "myobject.h"

#include <iostream>


MyObject::MyObject()
    : KPropertyPage()
{}

MyObject::MyObject(const QString &msg)
    : KPropertyPage(msg)
{}

MyObject::~MyObject()
{
    KPropertyPage::reset();
}


void MyObject::sayMessage(const std::string& test) {
    std::cout << test << std::endl;
    KPropertyPage::bla();
}
